paddle/capi/examples/model_inference/multi_thread/main_gpu.c

```diff
@@ -0,0 +1,106 @@
+#include <paddle/capi.h>
+#include <pthread.h>
+#include <time.h>
+#include "../common/common.h"
+
```

Comment:
Add some comments about this example. For example, this is an inference implementation where multiple threads share a GPU.

Reply:
Done.